src/opennlp/nlp.clj

```diff
@@ -25,13 +27,13 @@
   (if (seq span-col)
     (if (string? data)
       (seq
-        (Span/spansToStrings
-          #^"[Lopennlp.tools.util.Span;" (into-array span-col)
-          ^String data))
+       (Span/spansToStrings
```

Comment:
Can you undo these whitespace changes? They aren't needed for this PR

Reply:
I'm used cljfmt

Reply:
I used cljfmt